### PR TITLE
test(client): make it possible to pass extra args to jest in new setup

### PR DESCRIPTION
**Pull request by aqrln**
~~Make `test:functional` command run `jest --verbose --config=tests/functional/jest.config.js` instead of `pnpm run test:functional:code && pnpm run test:functional:types`.~~

~~This has the same effect by default, but allows to append extra CLI arguments after `pnpm run test:functional --`.~~

Add new `test:functional:with-args` command.

The previous approach wouldn't have worked because the default command would've been broken because there would be no guarantee tests would be generated before typechecking.

### CODE REVIEW

**Review by Jolg42 — APPROVED** [review]
👍🏼

File: packages/client/package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "test:functional": "pnpm run test:functional:code && pnpm run test:functional:types",
     "test:functional:code": "jest --verbose --config=tests/functional/jest.config.js --testPathIgnorePatterns typescript",
     "test:functional:types": "jest --verbose --config=tests/functional/jest.config.js -- typescript",
+    "test:functional:with-args": "jest --verbose --config=tests/functional/jest.config.js",
```
Comment (Jolg42) [inline packages/client/package.json:49]:
```suggestion
    "test:functional:custom-args": "jest --verbose --config=tests/functional/jest.config.js",
```
Maybe

**Comment by millsp** [comment]
Related https://github.com/prisma/prisma/pull/13226#

**Comment by aqrln** [comment]
Closing in favor of https://github.com/prisma/prisma/pull/13226

**Comment by aqrln** [comment]
I didn't know `pnpm test:functional:code -- -- mytest` worked in the first place 😄

**Pull request closed without merging**